charts/chronograf/README.md

```diff
@@ -1,58 +1,52 @@
-# Chronograf
+# Chronograf Helm chart
 
-##  An Open-Source Time Series Visualization Tool
+[Chronograf](https://github.com/influxdata/chronograf) is an open-source web application used to visualize your monitoring data and easily create alerting and automation rules.
 
-[Chronograf](https://github.com/influxdata/chronograf) is an open-source web application built by the folks over at [InfluxData](https://influxdata.com) and written in Go and React.js that provides the tools to visualize your monitoring data and easily create alerting and automation rules.
-
-## QuickStart
-
-```bash
-helm repo add influxdata https://helm.influxdata.com/
-helm upgrade --install chronograf influxdata/chronograf --namespace monitoring
-```
-
-## Introduction
-
-This chart bootstraps a Chronograf deployment and service on a Kubernetes cluster using the Helm Package manager.
+The Chronograf Helm chart bootstraps a Chronograf deployment and service on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
 
 - Kubernetes 1.4+
-- PV provisioner support in the underlying infrastructure (optional)
+- (Optional) PV provisioner support in the underlying infrastructure
 
-## Installing the Chart
+## Install the chart
 
-To install the chart with the release name `my-release`:
+1. Add the InfluxData Helm repository:
 
-```bash
-helm upgrade --install my-release influxdata/chronograf
-```
+   ```bash
+   helm repo add influxdata https://helm.influxdata.com/
+   ``` 
+
+2. Run the following command, providing a name for your Chronograf Helm chart:
+
+   ```bash
+   helm upgrade --install my-chart-name influxdata/chronograf --namespace monitoring
+   ```
 
-The command deploys Chronograf on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+   This command deploys Chronograf on the Kubernetes cluster using the default configuration. To find parameters you can configure during installation, see [Configure the chart](#configure-the-chart).
 
-> **Tip**: List all releases using `helm list`
+  > **Tip**: To view all releases, run `helm list`.
```

Comment:
All InfluxData Helm chart releases?

Reply:
No, all releases of any chart installed via helm